src/swagger-specs/data-hygiene.yaml

```diff
@@ -0,0 +1,1485 @@
+      example: orgId=885737B25DC460C50A49411B@AdobeOrg
+    page:
+      name: page
+      in: query
+      schema:
+        type: integer
+      description: An integer that indicates which page of expirations to return.
+      example: 3
+    sandboxName:
+      name: sandboxName
+      in: query
+      schema:
+        type: string
+      description: >
+        Matches dataset expirations whose sandbox name exactly matches the argument. Defaults to the sandbox name in the 
+        request’s x-sandbox-name header. Use sandboxName=* to include dataset expirations from all sandboxes.
+      example: dev1
+    search:
+      name: search
+      in: query
+      schema:
+        type: string
+      description: >
+        Matches expirations where the specified string is an exact match for the expiration ID, or is contained in any of 
+        these fields: `author`, `display name`, `description`, `display name`, `dataset name`.
+      example: "John Q. Public"
+    status:
+      name: status
+      in: query
+      schema:
+        type: string
+      description: >
+        A comma-separated list of statuses. When included, the response matches dataset expirations whose current status is among 
+        those listed.
+      example: pending,canceled
+    updatedDate:
+      name: updatedDate
+      in: query
+      schema:
+        type: string
+      description: >
+        Like `createdDate` / `createdFromDate` / `createdToDate`, but matches against a dataset expiration’s update time instead 
+        of creation time. An expiration is considered updated on every edit, including when it is created, cancelled, or executed.
+      example: '2022-01-01'
+    updatedToDate:
+      name: updatedToDate
+      in: query
+      schema:
+        type: string
+      description: >
+        Like `createdDate` / `createdFromDate` / `createdToDate`, but matches against a dataset expiration’s update time instead 
+        of creation time. An expiration is considered updated on every edit, including when it is created, cancelled, or executed.
+      example: '2022-01-01'
+    updatedFromDate:
+      name: updatedFromDate
+      in: query
+      schema:
+        type: string
+      description: >
+        Like `createdDate` / `createdFromDate` / `createdToDate`, but matches against a dataset expiration’s update time instead 
+        of creation time. An expiration is considered updated on every edit, including when it is created, cancelled, or executed.
+      example: '2022-01-01'
+    productStatusDetails:
+      name: productStatusDetails
+      in: query
+      required: false
+      schema:
+        type: string
+      description: >
+        This parameter is only accepted when the expiration status is `executing`, or `completed`. Each element of the array contains 
+        a `productName` and a `productStatus`. The `productName` will be either `Data Management`,  `Identity Service`, or 
```

Comment:
With three values, it shouldn't be "either". I'd just phrase this as "Supported values for `productName` include...", or something like that.